core/src/main/java/org/infinispan/notifications/cachelistener/event/Event.java

```diff
@@ -9,7 +9,7 @@
  * @since 4.0
  */
 public interface Event<K, V> {
-   static enum Type {
+   enum Type {
```

Comment:
All the unneeded chnages are in the commit titled 'Fix IDE and compiler warnings' :) so are easy to ignore.